apps/dashboard/config/configuration_singleton.rb

```diff
@@ -291,6 +291,15 @@ def pinned_apps_menu_length
     config.fetch(:pinned_apps_menu_length, 6)
   end
 
+  # The configured launcher buttons
+  def launchers
+    config.fetch(:launchers, [])
+  end
+
+  def launchers_path
+    config.fetch(:launchers_path, [])
+  end
```

Comment:
I'm not sure if we should have multiple paths. I've found more options just gives more confusion. It's not clear what the benefit is when, at some point, we want to start enforcing things for security reasons (like reading any system file only if it's root:root 644 as an example).


I know we want to enable user defined config, which this may indicate, but I feel like user settings we'll solve when we solve.

Reply:
This feature to allow multiple external configuration locations was added because it was requested by Trey in our last meeting.

I can remove the feature or can make it a single location.
What do you think?

Reply:
Yea I think it was probably talk about user level conifgs which would be somewhere else, but we can defer that entirely.

I'd say let's just rely on the `lanchers` api above this so that admins configure the launchers in the existing file(s).

Reply:
OK - Will update

Reply:
Changes implemented